app/ancillaryFactory/src/components/NoAncillaryMessage.js

```diff
@@ -0,0 +1,15 @@
+// @flow strict
+
+import * as React from 'react';
+import { View, Text } from 'react-native'; // eslint-disable-line no-restricted-imports
```

Comment:
We have restricted imports for a reason, please use `Text` from `@kiwicom/mobile-shared` instead